.github/CODEOWNERS

```diff
@@ -2,9 +2,9 @@
 
 * @baentsch
 /oqsprov/oqs_hyb_kem.c @bhess
-/oqs-template/generate.yml @baentsch @bhess @feventura
-/CMakeLists.txt @baentsch @thb-sb
-/.circleci/config.yml @baentsch @thb-sb
-/.github/workflows @baentsch @thb-sb @jplomas
+/oqs-template/generate.yml @baentsch @bhess @feventura @alexrow
```

Comment:
GitHub complains about the CODEOWNERS file being invalid due to @alexrow not having write access, even though they are listed in the `oqs-providers-codeowners` team.

I checked with @ryjones about this, and apparently @alexrow has not yet accepted the invitation to join the team (and hence does not have write access).

Reply:
As discussed in a in person meeting with @dstebila and @SWilson4, this PR does not add alexrow to CODEOWNERS to allow merging this PR sooner. Instead, https://github.com/open-quantum-safe/oqs-provider/issues/531 has been created to track this.